src/mower_logic/src/mower_logic/behaviors/Behavior.h

```diff
@@ -86,7 +88,7 @@ class Behavior {
     requested_pause_flag |= reason;
   }
 
-  void start(mower_logic::MowerLogicConfig &c, std::shared_ptr<sSharedState> s) {
+  void start(mower_logic::MowerLogicConfig& c, std::shared_ptr<sSharedState> s) {
```

Comment:
This was done by `/pre-commit run --all-files`, I didn't touch it.

Reply:
That's strange 😲 I just modified the file as in your commit and ran that exact command, it complained and corrected it back to what it was before:
```
clang-format.............................................................Failed
- hook id: clang-format
- exit code: 1
- files were modified by this hook

src/mower_logic/src/mower_logic/behaviors/Behavior.h
====================
--- original

+++ formatted

@@ -83,7 +83,7 @@

     requested_pause_flag |= reason;
   }

-  void start(mower_logic::MowerLogicConfig& c, std::shared_ptr<sSharedState> s) {
+  void start(mower_logic::MowerLogicConfig &c, std::shared_ptr<sSharedState> s) {
     ROS_INFO_STREAM("");
     ROS_INFO_STREAM("");
     ROS_INFO_STREAM("--------------------------------------");
```

Reply:
Oof, conflicting clang-format versions maybe?

```
$ clang-format --version
Debian clang-format version 14.0.6
```

Reply:
Exact same for me. Did you try `clang-format -n src/mower_logic/src/mower_logic/behaviors/Behavior.h`? It emits two warnings for me with `MowerLogicConfig& c`, and changes it to `MowerLogicConfig &c` with `-i` instead of `-n`.

Reply:
`clang-format -n src/mower_logic/src/mower_logic/behaviors/Behavior.h` gives no warning for me :facepalm: